.gitignore

```diff
@@ -41,6 +41,7 @@ _UpgradeReport_Files/
 ipch/
 *.sdf
 *.opensdf
+node.VC.opendb
```

Comment:
No opinion from me on whether or not it should be added, but if it should, then let's do it in a separate PR. It's unrelated to the main change here.